src/main/scala/gitbucket/core/view/helpers.scala

```diff
@@ -34,29 +34,37 @@ object helpers extends AvatarImageProvider with LinkConverter with RequestCache
   /**
    * Format java.util.Date to "x {seconds/minutes/hours/days/months/years} ago"
    */
-  def datetimeAgo(date: Date): String = {
-    val duration = new Date().getTime - date.getTime
-    timeUnits.find(tuple => duration / tuple._1 > 0) match {
-      case Some((unitValue, unitString)) =>
-        val value = duration / unitValue
-        s"${value} ${unitString}${if (value > 1) "s" else ""} ago"
-      case None => "just now"
+  def datetimeAgo(date: Date)(implicit context: Context): String = {
```

Comment:
It's nice to apply that the result of this method (and also `datetimeAgoRecentOnly`) is changed by `context.settings.relativeTime` to Scaladoc comment.